openhtf/util/units/__init__.py

```diff
@@ -0,0 +1,1170 @@
+# coding: iso-8859-1
```

Comment:
There isn't a second place for this, unless you're talking about the proto? The source is that xls file, but he's not checking that in.